src/ngraph_encapsulate_op.cc

```diff
@@ -34,6 +34,7 @@
 #include "ngraph_mark_for_clustering.h"
 #include "ngraph_utils.h"
 
+#include "ngraph/runtime/backend.hpp"
 #include "ngraph/runtime/interpreter/int_backend.hpp"
```

Comment:
In that case `#include "ngraph/runtime/interpreter/int_backend.hpp"` is not needed any more?